api/src/test/java/org/openmrs/module/operationtheater/api/dao/SurgicalBlockDAOTest.java

```diff
@@ -459,4 +459,28 @@ public void shouldReturnOverllappingSurgicalBlocksAsZeroWhenNewSurgicalBlockEndi
 		
 		assertEquals(0, surgicalBlocks.size());
 	}
+	
+	@Test
+	public void shouldReturnSurgicalWhichAreAcrossMultipleDaysIfWePassActiveAsTrue() throws Exception {
```

Comment:
@tarunkumar-tw do you mean SurgicalBlocks in the test name? if yes, can you change it accordingly

Reply:
Done